src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -676,6 +678,10 @@ R"(#include <stdio.h>
                             v->m_intent == ASRUtils::intent_return_var ) {
                             d += ";\n";
                         }
+                        if (ASR::is_a<ASR::SymbolicExpression_t>(*v->m_type)) {
+                            std::string v_m_name = v->m_name;
+                            d += indent + "basic_new_stack(" + v_m_name + ");\n";
```

Comment:
@anutosh491 do variables allocated using `basic_new_stack()` need to be deallocated (probably using `basic_free_stack()`)? I think it need not be done in this PR, but I am curious to know.

Reply:
Yes , yes . I've had this discussion with @certik and we even went over how we could maybe solve it in the future . We shall be addressing this !

Reply:
Resolving .